### PR TITLE
Add the command `hub issue view <NUMBER>`

### PR DESCRIPTION
**Pull request by royels**
This commit adds the subcommand hub issue view, which displays an issue as well as
lists its comments underneath.

Addresses issue #1865

### CODE REVIEW

**Comment by royels** [comment]
Done! Let me know if there's anything else I can do.

File: features/issue.feature [hunk 1]
```diff
@@ -530,3 +530,100 @@ Feature: hub issue
...
+        :number => 102,
+        :body => "I want this feature",
+        :title => "Feature request for hub issue view",
+        :created_at => "2017-04-14T16:00:49Z",
+        :user => { :login => "royels" }
+     }
+      get('/repos/github/hub/issues/102/comments') {
+      status 404
+      json [{
+              :id => 1,
```
[the remark this answers is not in the document]
Comment (royels) [reply features/issue.feature]: done

File: commands/issue.go [hunk 2]
```diff
@@ -371,6 +380,69 @@ func formatIssue(issue github.Issue, format string, colorize bool) string {
...
+
+	gh := github.NewClient(project.Host)
+
+	var issue = &github.Issue{}
+	issue, err = gh.FetchIssue(project, issueNumber)
+	utils.Check(err)
+
+	var closed = ""
+	if issue.State != "open" {
+		fmt.Println(issue.State)
```
Comment (mislav) [inline commands/issue.go]: extra `fmt.Println`?

**Comment by mislav** [comment]
I've tweaked the code style and added extra `--format` capabilities to `hub issue show` and `hub release show` as well. Thank you for your contribution!

**Pull request merged**